content/influxdb/cloud-iox/query-data/flux-sql.md

```diff
@@ -0,0 +1,522 @@
+Each row has a column for each queried field, tag, and time.
+In the context of Flux, SQL results are ungrouped. This is important to understand
+if you further process SQL results with Flux.
+
+The [example query above](#use-sql-and-flux-together) returns:
+
+{{% influxdb/custom-timestamps %}}
+
+|  co |  hum | room        | temp | time                 |
+| --: | ---: | :---------- | ---: | :------------------- |
+|   0 | 35.9 | Kitchen     |   21 | 2022-01-01T08:00:00Z |
+|   0 | 36.2 | Kitchen     |   23 | 2022-01-01T09:00:00Z |
+|   0 | 36.1 | Kitchen     | 22.7 | 2022-01-01T10:00:00Z |
+|   0 |   36 | Kitchen     | 22.4 | 2022-01-01T11:00:00Z |
+|   0 | 35.9 | Living Room | 21.1 | 2022-01-01T08:00:00Z |
+|   0 | 35.9 | Living Room | 21.4 | 2022-01-01T09:00:00Z |
+|   0 |   36 | Living Room | 21.8 | 2022-01-01T10:00:00Z |
+|   0 |   36 | Living Room | 22.2 | 2022-01-01T11:00:00Z |
+
+{{% /influxdb/custom-timestamps %}}
+
+## Process SQL results with Flux
+
+With your base data set returned from `iox.sql()`, you can further process your
+data with Flux to perform actions such as complex data transformations, alerting,
+HTTP requests, etc.
+
+{{% note %}}
+#### For the best performance, limit SQL results
+
+All data returned by `iox.sql()` is loaded into memory and processed there.
+To maximize the overall performance of your Flux query, try to return as little
+data as possible from your SQL query.
+This can by done by downsampling data in your SQL query or by limiting the
+queried time range.
+{{% /note %}}
+
+1. [Group by tags](#group-by-tags)
+1. [Rename the `time` column to `_time`](#rename-the-time-column-to-_time)
+1. [Unpivot your data](#unpivot-your-data)
+
+### Group by tags
+
+The Flux `from()` functions returns results grouped by measurement, tag, and field key
+and much of the Flux language is designed around this data model.
+Because SQL results are ungrouped, to structure results the way many Flux
+functions expect, use [`group()`](/flux/v0.x/stdlib/universe/group/) to group by
+all of your queried tag columns.
+
+{{% note %}}
+Measurements are not stored as a column in the InfluxDB IOx storage engine and
+are not returned by SQL.
+{{% /note %}}
+
+The [Get started sample data](#sample-data) only includes one tag: `room`.
+
+```js
+import "experimental/iox"
+
+iox.sql(...)
+    |> group(columns: ["room"])
+```
+
+_`group()` does not guarantee sort order, so you likely need to use
+[`sort()`](/flux/v0.x/stdlib/universe/sort/) to re-sort your data time **after**
+performing other transformations._
+
+### Rename the `time` column to `_time`
+
+Many Flux functions expect or require a column named `_time` (with a leading underscore).
+The IOx storage engine stores each point's timestamp in the `time` column (no leading underscore).
+Depending on which Flux functions you use, you may need to rename the `time`
+column to `_time`.
+YOu can do this in your SQL query with an `AS` clause or in Flux with the
```

Comment:
Y0u -> You

I am wondering if we reiterate here that for the best performance, they should use SQL `AS` rather than rename()

Reply:
Added a recommended note after "an `AS` clause".